src/yaml-support/yaml-schema.ts

```diff
@@ -0,0 +1,208 @@
+import * as _ from 'lodash';
+import Uri from 'vscode-uri';
+import * as vscode from 'vscode';
+import { yamlLocator } from "./yaml-locator";
+import {
+    VSCODE_YAML_EXTENSION_ID, KUBERNETES_SCHEMA, KUBERNETES_GROUP_VERSION_KIND, GROUP_VERSION_KIND_SEPARATOR,
+    KUBERNETES_SCHEMA_FILE
+} from "./yaml-constant";
+import * as util from "./yaml-util";
+
+// the schema for kubernetes
+const _definitions = {};
+
+// load the kubernetes schema and make some modifications to $ref node
+export function loadSchema(schemaFile: string) {
+    const schemaRaw = util.loadJson(schemaFile);
+    const definitions = schemaRaw.definitions;
+    const toFixed = [];
```

Comment:
does convertRefToSchemaURIs name the action?

Reply:
Yes - as a reader, it would certainly set me on the right track to understanding.  For the array, I'd prefer a noun or adjective, but if it's hard to come up with a good one, then at least this would point me in the direction of what the array was for.